### ISSUE TITLE
Add standardized notes for services that require contacting customer support

### ISSUE THREAD

**Issue by WebSnke** [jdm-contrib/jdm#1742]
The services that require contacting customer support have similar notes.

```
"notes": "Account deletion requires contacting Customer Support.",
"notes_tr": "Hesabınızın silinmesi için müşteri hizmetleri ile iletişime geçmeniz gerekir.",
"notes_fr": "Pour supprimer votre compte contactez le support.",
"notes_it": "La cancellazione account richiede di contattare il servizio clienti.",
"notes_de": "Account-Löschung erfordert es den Kundensupport zu kontaktieren",
"notes_pt_br": "Para deletar a sua conta contate a assistência ao cliente",
"notes_cat": "Per esborrar el seu compte contacti amb l'assistència al client",
"notes_es": "Para borrar tu cuenta contacta con asistencia al cliente"
```

All of them have the "email" property. That's why I think that all entries that the "email" property should add these notes automatically. This would make it easier to add more translations and would save dozens of lines in the JSON file.

Let me know what you think.

**Comment by tupaschoal** [comment]
I'm torn on the default notes, because I want to make the submission process as easy as possible and maybe having to tell people to remove the notes because the default one covers it might be too cumbersome, what do you think?

**Comment by WebSnke** [comment]
> I'm torn on the default notes, because I want to make the submission process as easy as possible and maybe having to tell people to remove the notes because the default one covers it might be too cumbersome, what do you think?

I don't think that it would be combersome to explain to new contributors and it would improve localisation and consistency.

**Comment by tupaschoal** [comment]
Ok

**Comment by WebSnke** [comment]
What if we add these standardised notes as the default notes for the services with a difficulty of "hard"?

**Comment by tupaschoal** [comment]
I think that's fair. We could have a default note saying it's customer support and maybe another that is combined on the first one to stress that it is through email, if such property is set?

**Comment by aura-lsprog-86** [comment]
> I think that's fair. We could have a default note saying it's customer support and maybe another that is combined on the first one to stress that it is through email, if such property is set?

I think this might be appropriate, especially for services that, having available easy deletion mode, nonetheless offer means to contact their Tech Support team in case the easy deletion flow fails for some reason. For example, [omegaUp](https://omegaup.com/) is one of those services with such a provision.

**Comment by OphirBleicher** [comment]
I see this is already implemented. In translations where there is a `defaultnote_email` field in the translation JSON and no notes are attached to the site in `sites.json`, the default email note appears. When there isn't a `defaultnote_email` field, the site fallbacks to the English `defaultnote_email`.
I'm creating a pull request to inform new contributors in CONTRIBUTING.md.

**Comment by OphirBleicher** [comment]
I've created PR #2974 to inform new contributors that there are default notes for hard difficulty sites.

**Comment by tupaschoal** [comment]
> I've created PR [#2974](https://github.com/jdm-contrib/jdm/pull/2974) to inform new contributors that there are default notes for hard difficulty sites.

That's not 100% true, we will only add a default note when:
- Notes are not set and difficulty is easy
- Notes are not set and an `e-mail` is set.

So if one leaves the `notes` unset, but doesn't set an `e-mail`, nothing will show up

**Comment by OphirBleicher** [comment]
I informed contributors about the standardized notes in PR #2974 and I think this issue can be closed.

**Comment by tupaschoal** [comment]
No, it's not fully solved. The default notes still doesn't apply if the difficulty is set to hard, but doesn't have an associated e-mail (if for example you have to  contact through a ticketing system of their own).

The full solution of this issue would implement that + it would clean-up all hard entries from default notes that basically say the same thing as the default would.

**Comment by OphirBleicher** [comment]
PR #2995 fixed this issue and was merged, so I think this can be closed.

**Comment by tupaschoal** [comment]
> PR [#2995](https://github.com/jdm-contrib/jdm/pull/2995) fixed this issue and was merged, so I think this can be closed.

Actually no, like I had said above, the full solution would also clean-up the file to remove entries that are redundant with the newly introduced default notes. I made such an attempt at #3011, but there is likely more that can be done if we also consider the pre-existing defaults.
